factfeel_client.py

```diff
@@ -1,12 +1,16 @@
 # imports
 from phue import Bridge
+import speech_recognition as sr
```

Comment:
I reorganized the imports based on 3rd party apps (on top), core anaconda libraries (middle) and base python libraries (bottom)